systemdev/testing-locally.rst

```diff
@@ -125,26 +131,77 @@ In both cases the chroot will be created by running the following command::
 
     sudo sbuild-createchroot --components=main,restricted,universe --extra-repository="deb http://archive.ubuntu.com/ubuntu/ ${chroot_distro}-updates main restricted universe" --include=ccache "${chroot_distro}" "${chroot_base}" http://archive.ubuntu.com/ubuntu/
 
-A chroot for cross-building arm64 packages on an amd64 host can e.g. be created under the directory ``/srv/chroot/ubports-${chroot_distro}-arm64`` using::
+The UBports package repository needs to be added using::
+
+    wget 'http://repo.ubports.com/keyring.gpg' -O - | sudo tee "${chroot_base}/usr/share/keyrings/ubports-keyring.gpg" >/dev/null
+    printf 'deb [signed-by=/usr/share/keyrings/ubports-keyring.gpg] %s %s main\n' "${chroot_repo}" "${chroot_distro}" | sudo tee "${chroot_base}/etc/apt/sources.list.d/ubports.list" >/dev/null
+
+Synchronizing package index files and subsequent package upgrades can be performed using::
+
+    sbuild-update -u -d ${chroot_distro}
+    
+A build can be started from inside the debianized package source directory using::
+
+    sbuild -d ${chroot_distro}
```

Comment:
in order to build `l-a-l` 2 packages from the UBports repo need to be installed on the host machine.

This are the stepts if you want to try them on your end:

`// add the UBports repo in the HOST machine
wget 'http://repo.ubports.com/keyring.gpg' -O - | sudo tee /usr/share/keyrings/ubports-keyring.gpg" >/dev/null
printf 'deb [signed-by=/usr/share/keyrings/ubports-keyring.gpg] %s %s main\n' "${chroot_repo}" "${chroot_distro}" | sudo tee "/etc/apt/sources.list.d/ubports.list" >/dev/null
sudo apt update
sudo apt install click-dev gobject-introspection`

Reply:
Shouldn't it be part of the instruction then?

Reply:
> Shouldn't it be part of the instruction then?

it is specific for the module which you build , thats why i didnt added it , but we can mention that this is needed for some modules

Reply:
Is it always these two packages in case something additional is needed? Is there a way to tell which one is needed for which app? Would be great if the docs enabled contributors to figure it out themselves.

Reply:
> Is it always these two packages in case something additional is needed? Is there a way to tell which one is needed for which app? Would be great if the docs enabled contributors to figure it out themselves.

unfortunatelly i dont know, so far i only tested it with `l-a-l`.

Reply:
You could add a small section with these instructions as an example how to install additional dependencies. It seems like I needed that for lomiri system settings, too, at least.

Reply:
got will do

Reply:
> You could add a small section with these instructions as an example how to install additional dependencies. It seems like I needed that for lomiri system settings, too, at least.

done

added `UBports packages` section